### PR TITLE
chore: Update FAQ

### PR DESCRIPTION
**Pull request by wsdt**

### CODE REVIEW

**Review by jyellick — APPROVED** [review]
LGTM, but I think line 58/60 needs to be amended as well, as there are now at least a couple exchanges that integrate directly with Boba

File: .gitbook/FAQ.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: Frequently asked questions
 <figure><img src="./assets/What is Boba Network.png" alt=""><figcaption></figcaption></figure>
 
 
-[Boba](https://boba.network) is a compute-focused Layer 2 (L2) solution built on top of the Layer 1 blockchain, [Ethereum](https://ethereum.org/en). Ethereum is similar to L1 blockchains like Avalanche, BNB, and Moonbeam, and acts as a base blockchain platform that executes all on-chain transactions. Boba scales and augments the core _compute_ capabilities of Ethereum, reducing gas fees and improving transaction throughput - while retaining the security guarantees of Ethereum.
+[Boba](https://boba.network) is a compute-focused Layer 2 (L2) solution built on top of the Layer 1 blockchain, [Ethereum](https://ethereum.org/en) and [BSC](https://www.bnbchain.org/en/bnb-smart-chain). Ethereum is similar to L1 blockchains like Avalanche, BNB, and Moonbeam, and acts as a base blockchain platform that executes all on-chain transactions. Boba scales and augments the core _compute_ capabilities of Ethereum/BSC, reducing gas fees and improving transaction throughput - while retaining the security guarantees of its underlying network.
```
Comment (jyellick) [inline .gitbook/FAQ.md]: Not sure how critical it is, but I believe "BSC" is now "BNB Chain"

**Comment by wsdt** [comment]
> LGTM, but I think line 58/60 needs to be amended as well, as there are now at least a couple exchanges that integrate directly with Boba

incorporated both comments, thx!

**Review by souradeep-das — APPROVED** [review]

File: .gitbook/FAQ.md [hunk 2]
```diff
@@ -49,19 +51,19 @@ If your wallet does not support this feature, you will have to connect manually.
...
+You can choose any of these two methods: [the **classical bridge**, or the **light bridge**](contents/boba-basics/bridge-basics/).
 
 ## Can I cancel a withdrawal after it has been submitted?
 
 _No_, withdrawals currently cannot be cancelled once submitted.
 
 ## Can I transfer directly from Boba to a centralized exchange?
 
-Only if the centralized exchange supports Boba Network (at this time none of them do). Hence, you have to bridge your assets into a network they do support, such as L1.
+Only if the centralized exchange supports Boba Network. Hence, you have to bridge your assets into a network they do support, such as L1.
```
Comment (jyellick) [inline .gitbook/FAQ.md]:
```suggestion
Only if the centralized exchange supports Boba Network. For exchanges that do not directly support Boba Network, you can still bridge your assets into a network they do support (such as Eth Mainnet or BNB Chain).
```

Doesn't need to be this text exactly, but the 'Hence', doesn't read correctly to me.

**Pull request merged**